app/src/main/java/com/example/remindmelater/MainActivity.kt

```diff
@@ -169,16 +175,19 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
 
         @Composable
         fun ReminderListItem() {
-            Column() {
```

Comment:
Not being used but clearly a non-trivial implementation that will be used later.